### DISCUSSION TITLE
New Column

### DISCUSSION THREAD

**Discussion by dhruvi-alphabin** [theoephraim/node-google-spreadsheet#621] [General]
How can I add new column?

**Comment by theoephraim** [comment]
if you mean inserting more columns into the sheet, https://theoephraim.github.io/node-google-spreadsheet/#/classes/google-spreadsheet-worksheet?id=fn-insertdimension

If you mean adding a new header for the row-based api, you can use https://theoephraim.github.io/node-google-spreadsheet/#/classes/google-spreadsheet-worksheet?id=fn-setheaderrow
